package.json

```diff
@@ -1,5 +1,9 @@
 {
   "private": true,
+  "packageManager": "pnpm@8.15.3",
+  "engines": {
+    "pnpm": "^8.15.0"
```

Comment:
When `corepack` is not enabled, fallback to `npm`'s built-in "engine" enforcement.